app/views/albums/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row">
+  <div class="col-lg-12">
+    <table class="table table-hover">
+      <tbody>
+        <tr>
+          <th>Ranking</th>
+          <th>Name</th>
+          <th>Upvote</th>
+        </tr>
+        <% @albums.each do |album| %>
+          <tr>
+            <td><%= album.rank %></td>
+            <td><%= link_to album.name, album_path(album.id) %></td>
+            <td><%= button_to "Upvote", upvote_albums_path(album.id), :class => "btn btn-default" %></td>
```

Comment:
I recommend being consistent with your key/value syntax. Here, you use `:class => "...."` but down just a few lines, you use `class: "...."`